### ISSUE TITLE
Upgrade to HOOBS 4 unavailable

### ISSUE THREAD

**Issue by VulcanB13** [hoobs-org/HOOBS#1425]
Options to upgrade to HOOBS 4 is unavailable. Read the documentation and posts on the FB channel... 
When I click on the dialog box to check for updates, the screen goes to the system software screen that displays current config!
What's missing and what needs to be done? Any help is appreciated
Here is my current config:

HOOBS Version | 3.3.6
Node Version | 12.14.1

**System**
Manufacturer | Raspberry Pi Foundation
-- | --
Model | Raspberry Pi 4 Model B Rev 1.2
Version | b03112

**File System**
/ | Used 30.36%
-- | --
/boot | Used 20.55%

**Comment by mkellsy** [comment]
There are things to check. This doc should help you out.
https://support.hoobs.org/docs/60e62a926c19a6a5ca8f4f42

**Comment by VulcanB13** [comment]
> 
> 
> There are things to check. This doc should help you out.
> https://support.hoobs.org/docs/60e62a926c19a6a5ca8f4f42

I've already checked that document before reaching out for support. That works when HOOBS is on v 3.3.7. My HOOBS is stuck on 3.3.6 with no prompts to upgrade. What are the options besides a clean install of HOOBS 4?? I don't want to re-install, because it will mean re-doing all my mappings and re association on the phones!

**Comment by askovi** [comment]
Try connecting via ssh with user hoobs and password hoobsadmin and running this command to update:

`sudo npm install -g @hoobs/hoobs`

**Comment by VulcanB13** [comment]
> 
> 
> Try connecting via ssh with user hoobs and password hoobsadmin and running this command to update:
> 
> `sudo npm install -g @hoobs/hoobs`

Tried it & it gave me the below line. Didn't take the hoobsadmin password and didn't take my password for Hoobs!
**_[sudo] password for nobody:_**

**Comment by mkellsy** [comment]
The password is hoobsadmin
That command is for 3.3. 4.0 has moved to APT.
The nobody username has me worried. You might be better off flashing a fresh 4.0 image and starting over.
https://support.hoobs.org/downloads

**Comment by VulcanB13** [comment]
> 
> 
> The password is hoobsadmin
> That command is for 3.3. 4.0 has moved to APT.
> The nobody username has me worried. You might be better off flashing a fresh 4.0 image and starting over.
> https://support.hoobs.org/downloads

I went ahead and downloaded the v 4, flashed the SD card & installed the plugins.. So far so good. Here is what I'm facing:
Installed three plugins in individual bridge without any issues & am able to add them to my iphone and control the devices as well:
a. Rachio Platform
b. Harmonyhub plugin
c. MyQ bridge

When I go to each bridge, I see the QR code for the plugin and they show up on the iPhone Home app and control works well.
Here is the challenge - the plugin for the Hubitat MakerAPI doesn't seem to want to run nor does it show the QR code, or display the devices which were working well in HOOBS v 3!!!
When I go to the dashboard, I see a box which says Bridges 3 Running, 1 Down.
I go to the Hubitat Maker API bridge & the plugin has a button under Pairing to Start. Nothing happens when I Click on the Start button. I'm supposed to get options to enter the URL information for Hubitat - nothing happens.

I did a little bit more of troubleshooting -> Clicked on the Gear Icon on the bottom left, Clicked on "Advanced" & under the "Hubitat Makerapi Bridge" tab, I edited the configuration text to contain the same text from my previous configuration & clicked on "Save" @ the bottom.... this resulted in the HOOBS graphic flowing thru in the middle of the screen - it stays that way without a change! Going back to the config, I see that the config for the Makerapi is back to default.
I checked the logs and this is what I see: Any help is appreciated

8/9/2021, 5:34:23 PMhubitatmakerapibridge stopping
8/9/2021, 5:34:28 PMHubitat MakerAPI Bridge starting
8/9/2021, 5:48:33 PMHubitat Makerapi Bridge 2 starting
8/9/2021, 5:54:12 PMHubitat Makerapi Bridge 2Configuration change
8/9/2021, 5:54:12 PMhubitatmakerapibridge2 stopping
8/9/2021, 5:54:12 PMHubitat Makerapi Bridge 2WarningNo plugins installed.
8/9/2021, 5:54:12 PMHubitat Makerapi Bridge 2Bridge is running on port 51866.
8/9/2021, 5:54:17 PMHubitat Makerapi Bridge 2 starting
8/9/2021, 5:56:48 PMhubitatmakerapibridge stopping
8/9/2021, 5:56:46 PMHubitat Makerapi Bridge 2Loaded plugin 'homebridge-hubitat-makerapi'
8/9/2021, 5:56:46 PMHubitat Makerapi Bridge 2Loading 1 platforms...
8/9/2021, 5:56:47 PMHubitat Makerapi Bridge 2Warningunhandled rejection: TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received undefined
8/9/2021, 6:13:30 PMhubitatmakerapibridge2 stopping
8/9/2021, 6:13:36 PMHubitat Makerapi Bridge 2 starting
8/9/2021, 6:13:38 PMHubitat Makerapi Bridge 2Loaded plugin 'homebridge-hubitat-makerapi'
8/9/2021, 6:13:38 PMHubitat Makerapi Bridge 2Loading 1 platforms...
8/9/2021, 6:13:39 PMHubitat Makerapi Bridge 2Warningunhandled rejection: TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received undefined
